src/main.py

```diff
@@ -1,16 +1,28 @@
-# This is a sample Python script.
+import unittest
+
+
+class ReturnClass:
+    def return_number(self):
+        return 897
```

Comment:
Leave the main as it was. Create second file in src named `ExampleClass` and Rename this class to the same name as file for example. filename = `example_class.py` and class name `ExampleClass`

About function call it `exampleFunction` Check [pepe 0008](https://peps.python.org/pep-0008/) to understand how you should name Classes, functions, variables etc